CMakeLists.txt

```
@@ -113,7 +113,7 @@ include(px4_parse_function_args)
 include(px4_git)
 
 execute_process(
-	COMMAND git describe --exclude ext/* --tags --match "v[0-9]*"
+	COMMAND git describe --exclude ext/* --match v[0-9]*.[0-9]*.[0-9]* --abbrev=0 --tags
```

Why `--abbrev=0`? I think we specifically want it to include the SHA when it's not directly on a tag right?

```suggestion
	COMMAND git describe --exclude ext/* --match v[0-9]*.[0-9]*.[0-9]* --tags
```


For example

``` bash
% git describe 
v1.15.0-alpha1-707-g84c4a55a03

% git describe --abbrev=0
v1.15.0-alpha1
```